src/seedu/addressbook/data/person/Name.java

```diff
@@ -37,10 +37,12 @@ public static boolean isValidName(String test) {
     }
 
     /**
-     * Retrieves a listing of every word in the name, in order.
+     * Retrieves a listing of every word in the name in lowercase, in order.
      */
-    public List<String> getWordsInName() {
```

Comment:
I would suggest not to remove the existing method since the behavior of the existing method and new method are different. You are safe to remove it if this method is not required at all in the future and not used in any other places.